src/core/ext/filters/client_channel/resolver/google_c2p/google_c2p_resolver.cc

```diff
@@ -173,7 +173,8 @@ void GoogleCloud2ProdResolver::StartLocked() {
           absl::StatusOr<std::string> result) mutable {
         resolver->work_serializer_->Run(
             [resolver, result = std::move(result)]() {
-              resolver->IPv6QueryDone(result.ok());
+              resolver->IPv6QueryDone(result.ok() ? std::move(result).value()
```

Comment:
Why don't we just change this to `result.ok() && !result->empty()`?  There's no need to actually propagate the resulting string anywhere, since all we care about is whether or not it's empty.  Seems like this would just be a one-line change.

Reply:
good idea, done. I was initially going for consistency with the way the zone query is handled. But your suggestion is simpler.